scripts/copy_rootstrap.sh

```diff
@@ -23,9 +23,19 @@ if [[ -z $TIZEN_SDK ]]; then
   fi
 fi
 
-rootstrap=$TIZEN_SDK/platforms/tizen-$version/iot-headed/rootstraps/iot-headed-$version-device.core
+if [ "$version" == "8.0" ]
+then
```

Comment:
For consistency and portability (`==` is not posix standard):
```suggestion
if [ "$version" = "8.0" ]; then
```

Reply:
done